examples/bri-3/package.json

```diff
@@ -36,8 +36,10 @@
     "@nestjs/passport": "^9.0.0",
     "@nestjs/platform-express": "^9.0.0",
     "@prisma/client": "^4.4.0",
+    "bcrypt": "^5.1.0",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.13.2",
+    "crypto": "^1.0.1",
```

Comment:
@biscuitdey This library seems to be deprecated and built into node modules. Its likely unnecessary